sys/net/gnrc/network_layer/ipv6/nib/nib.c

```diff
@@ -1397,7 +1397,8 @@ static bool _resolve_addr(const ipv6_addr_t *dst, gnrc_netif_t *netif,
         return false;
     }
 
-    if (IS_ACTIVE(CONFIG_GNRC_IPV6_NIB_ARSM)) {
+    /* don't send multicast neighbor solicitation on 6lo */
+    if (!gnrc_netif_is_6lo(netif)) {
```

Comment:
Also: shouldn't the test rather be for `gnrc_netif_is_6ln` (is 6LoWPAN node (6LN) as defined in RFC 6775) instead of just any `gnrc_netif_is_6lo` (any generic 6LoWPAN interface which might not necessarily use 6Lo-ND as specified in RFC 6775)